systemtest/src/test/java/io/strimzi/systemtest/BaseST.java

```diff
@@ -643,6 +646,22 @@ void verifyLabelsForRoleBindings(String clusterName, String appName) {
         );
     }
 
+    protected NetworkPolicy applyNetworkPolicySettings(String clientNamespace, List<String> namespaces) {
+        NetworkPolicy networkPolicy = null;
+
+        for (String namespace : namespaces) {
+            if (Environment.ALLOW_NETWORK_POLICIES.equals("true")) {
+                networkPolicy = KubernetesResource.allowNetworkPolicies(namespace);
+            } else {
+                networkPolicy = KubernetesResource.denyNetworkPolicies(namespace);
+            }
+        }
+
+        LOGGER.info("NetworkPolicy successfully set to: {} for namespace: {}", Environment.ALLOW_NETWORK_POLICIES, clientNamespace);
+
+        return networkPolicy;
+    }
+
```

Comment:
This one i would move to KubeClusterResource class what do you think @Frawless ?

Reply:
Yes, I already suggested it I think (or at least I wanted:D )